app/features/font.test.js

```diff
@@ -30,7 +30,7 @@ test('Can Be Deactivated', async t => {
   const { page } = t.context
 
   t.is(await getActiveTool(page), tool)
-  await page.evaluate(`document.querySelector('vis-bug').$shadow.querySelector('li[data-tool="padding"]').click()`)
+  await changeMode({ tool: 'padding', page })
```

Comment:
I know in general that [unit tests should favor being DAMP over being DRY, but I think in this case the helper function clarifies the test, since the detailed querySelector chain seems a little unrelated to the "Can Be Deactivated" tests](https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html).